Jenkinsfile

```diff
@@ -58,6 +59,30 @@ void unittests(String version) {
     }
 }
 
+void uploadPackagePyPI() {
+    node(Constants.DOCKERNODE) {
+        buildsCommon.cleanup()
+        checkout scm
+        withContainer(image: "python:3.6-buster", registry: '', inside: '-u root') {
+            try {
+                String prodOrTest = env.BRANCH_NAME == 'master' ? 'PROD' : 'TEST'
```

Comment:
What difference do the prod/test user make?

Reply:
PROD means it is master branch build , otherwise it is test branch and only relevant credentials place, upload will be done, if already exists, it will not upload. ie what PYPI is doing